Tests/Admin/AdminHelperTest.php

```diff
@@ -18,6 +18,9 @@
 
 class AdminHelperTest extends \PHPUnit_Framework_TestCase
 {
+    /**
+     * @covers Sonata\AdminBundle\Admin\AdminHelper::getChildFormBuilder
+     */
```

Comment:
If you as the writer of the test decide to make it `covered`, then add the annotation.

but `@covers a` is wrong in this case, but would not result in any coverage.

The order is:
* execute code
* if executed code should be covered by annotation -> mark the executed lines covered, otherwise not.